use/connect-to-the-physical-world/create-object.md

```diff
@@ -0,0 +1,200 @@
+Whatever you choose should be flat, and have a lot of visual detail. The less unique details it
+has, the worse it will track in AR. I decided to augment some book covers of classics on my shelf.
+
+![photo of old book covers](./images/create-object/book-covers.jpg)
+
+I took a photo with my phone of the Fahrenheit 451 cover, cropped it so that none of the
+background is showing, sent it to my computer, and exported it as a JPG file (it was originally
+a different format – needs to be JPG).
+
+Take some time to do the same. If you can't find something in your environment, you can also
+ download some sample images from the
+[Vuforia Engine developer portal](https://library.vuforia.com/content/vuforia-library/en/articles/Solution/sample-apps-target-pdfs.html)
+(click on any of the "Download" links for a PDF, and then export a page from the PDF as a JPG).
+
+Now that you have an image of your object, go back to the web interface and click on the "Add
+Target" button to show the "Drop Zone". Drag and drop your JPG file on top of the Drop Zone.
+
+![add target drop zone](./images/create-object/04-add-target-drop-zone.png)
+
+If successful, the page will refresh and the "Add Target" will now say "Edit Target" and show an
+icon of the image you uploaded. If it fails, make sure your image is an uncorrupted JPG file.
+Try with a different JPG if it still doesn't work.
+
+![new object entry initialized on server](./images/create-object/05-book-initialized.png)
+
+### Add a custom tool to your object
+
+At this point, your Vuforia Spatial Toolbox app will recognize this object, but there isn't any
+AR content on it yet. To give it some default content (that we can edit later) click on the "Add
+Tool" button, type in the name `interface`, and click the green "Add" button to create a new
+tool attached to that object.
+
+![add new tool to object](./images/create-object/06-book-add-tool.png)
+
+This has added a new custom tool to this object. This is a "local" tool that is unique to this
+object, rather than being a copy of a tool that is in the pocket. We'll explore what we can do
+with this tool a bit further down.
+
+![object with tool named interface](./images/create-object/07-book-with-tool-interface.png)
+
+At this point, everything is ready! Make sure your phone is connected to the same WiFi network as
+the computer whose Edge Server you added the object to. Open the Vuforia Spatial Toolbox app.
+
+Point your camera at whatever you took the photo of (or of the image on your computer screen, if
+you don't have the physical object on hand). You should see a blue square show up in AR! As you
+move the object around, the interface will move with it.
+
+![viewing the interface through the app](./images/create-object/08-spatial-toolbox-book-ui.gif)
+
+If you don't see the blue AR square when you look at your object, check out the
+[trouble-shooting](#troubleShooting) section at the bottom of this page.
```

Comment:
troubleshooting is more commonly one word

Reply:
thanks for proofreading! 🎉